{"name": "Проверка получения ошибки при попытке создания двух одинаковых курьеров", "status": "passed", "start": 1719523452752, "stop": 1719523453379, "uuid": "a7d4e825-84a8-45ec-8541-5e700f2f3055", "historyId": "22c637b75f2d190b46eaa771b7e30fb6", "testCaseId": "22c637b75f2d190b46eaa771b7e30fb6", "fullName": "TestsCourier.tests_create_courier.TestCreateCourier#test_register_two_same_login_return_error", "labels": [{"name": "parentSuite", "value": "TestsCourier"}, {"name": "suite", "value": "tests_create_courier"}, {"name": "subSuite", "value": "TestCreateCourier"}, {"name": "host", "value": "MacBook-Air-Anastasiia.local"}, {"name": "thread", "value": "7270-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestsCourier.tests_create_courier"}]}